docs/examples/index.md

```diff
@@ -54,14 +215,104 @@ object Uppercase extends IOApp.Simple:
     yield ()
 
 end Uppercase
+```
+
+@:choice(static)
+
+```scala 3 mdoc:reset
+import cats.effect.{IO, IOApp}
+import fs2.io.file.Path
+
+import shellfish.os.syntax.FilesOs
+
+object Uppercase extends IOApp.Simple:
+
+  val path      = Path("src/main/resources/quijote.txt")
+  val upperPath = Path("src/main/resources/quijote_screaming.txt")
+
+  def run: IO[Unit] =
+    for
+      file <- FilesOs.read(path)
+      _    <- FilesOs.write(upperPath, file.toUpperCase)
+    yield ()
+
+end Uppercase
+```
+
+@:choice(fs2)
+
+```scala 3 mdoc:reset
+import cats.effect.{IO, IOApp}
+
+import fs2.io.file.{Path, Files}
+
+
+object Uppercase extends IOApp.Simple:
+
+  val path      = Path("src/main/resources/quijote.txt")
+  val upperPath = Path("src/main/resources/quijote_screaming.txt")
+
+  def run: IO[Unit] =
+    Files[IO].readUtf8(path)
+      .map(_.toUpperCase)
+      .through(Files[IO].writeUtf8(upperPath))
+      .compile
+      .drain
 
+end Uppercase
 ```
 
+@:@
+
 ## Places
 
-Shellfish OS has compatibility for working with [scodec](https://scodec.org/)!
+Maybe you also want to work with binary files. In this example, we are going to create a binary file with a list of places. Because Shellfish has compatibility for working with [scodec](https://scodec.org/), we are going to use that library to encode and decode the binary file!
```

Comment:
```suggestion
Shellfish can handle binary files in custom serde formats since it includes [scodec](https://scodec.org/). In this example, we are going to create a binary file with a list of places.
```